### PR TITLE
migrate `examples` to `openApi` part5; affects [itunes jetbrains jitpack keybase lemmy luarocks maintenance openvsx]

### PR DESCRIPTION
**Pull request by chris48s**
Refs #9285

..and so begins the long slow process of working through these. The first batch I am working on is classes with:

- currently passing service tests
- only path params
- either one example, or two examples due to optional path params (e.g: `branch*`)
- no documentation string

These are the most trivial services to convert, requiring the least manual work.
I am going to submit PRs touching batches of about 15-20 files at a time as I work through these.
Then I'll start moving onto slightly more complicated cases.

### CODE REVIEW

File: services/jsdelivr/jsdelivr-hits-github.service.js [hunk 1]
```diff
@@ -6,17 +7,27 @@ export default class JsDelivrHitsGitHub extends BaseJsDelivrService {
     pattern: ':period(hd|hw|hm|hy)/:user/:repo',
```
Comment (chris48s) [inline services/jsdelivr/jsdelivr-hits-github.service.js:7]: TODO: enum

Comment (chris48s) [reply services/jsdelivr/jsdelivr-hits-github.service.js:7]: _sigh_ and also work out why the liberapay tests are failing (they pass locally)

Comment (chris48s) [reply services/jsdelivr/jsdelivr-hits-github.service.js]: I've just reverted this for now

**Comment by chris48s** [comment]
The tests for liberapay pass locally.
When I try to run them in GitHub actions, they fail.
It seems that when we call liberapay from GH actions, instead of a JSON response we get a 403 error with a javascript challenge (designed to prevent bots/scrapers/etc). You can see the output of `npm run badge https://img.shields.io/liberapay/gives/Changaco` invoked from GitHub actions here https://github.com/badges/shields/actions/runs/5730805389/job/15530375550

I'm not going to attempt to address that here. I've just removed it from the list of services we're testing against.

File: services/jetbrains/jetbrains-downloads.service.js [hunk 2]
```diff
@@ -29,15 +30,17 @@ export default class JetbrainsDownloads extends JetbrainsBase {
...
 
-  static examples = [
-    {
-      title: 'JetBrains plugins',
-      namedParams: {
-        pluginId: '1347',
+  static openApi = {
+    '/jetbrains/plugin/d/{pluginId}': {
+      get: {
+        summary: 'JetBrains plugin Downloads',
```
Comment (PyvesB) [inline services/jetbrains/jetbrains-downloads.service.js:36]:
```suggestion
        summary: 'JetBrains Plugin Downloads',
```

For consistency with the other JetBrains badge :)

**Review by PyvesB — APPROVED** [review]
Only a small nit, feel free to merge whenever 👍🏻

**Pull request merged**